disttask/framework/proto/task.go

```diff
@@ -86,9 +86,14 @@ type Subtask struct {
 	// SchedulerID is the ID of scheduler, right now it's the same as instance_id, exec_id.
 	// its value is IP:PORT, see GenerateExecID
 	SchedulerID string
-	StartTime   uint64
-	EndTime     time.Time
-	Meta        []byte
+	// StartTime is the time when the subtask is started.
+	// it's 0 if it hasn't started yet.
+	StartTime time.Time
+	// UpdateTime is the time when the subtask is updated.
+	// it can be used as subtask end time if the subtask is finished.
+	// it's 0 if it hasn't started yet.
```

Comment:
Since updteTime is end time， when it is 0， then not means not start.

Reply:
> Since updteTime is end time

it's not the end time, it can be used as subtask end time *ONLY IF* the subtask is finished.

Reply:
what I mean is “it's 0 if it hasn't started yet.” is not correct，will startsubtask update both starttime and updatetime during starttime？